### PR TITLE
updating eslint to global and 2.x

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers sooner the better

### CODE REVIEW

File: desktop/app/console-helper.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ let fileWritable = null
 if (logFile) {
   fs.access(logFile, fs.W_OK, err => {
     if (err && !fileDoesNotExist(err)) {
-      console.log(`Can't write to log file.`, err)
+      console.log("Can't write to log file.", err)
```
Comment (chrisnojima) [inline desktop/app/console-helper.js]: this'll be fixed soon: https://github.com/eslint/eslint/issues/5234

File: desktop/npm-helper.js [hunk 2]
```diff
@@ -3,7 +3,22 @@ import path from 'path'
 import child_process, {execSync} from 'child_process'
 import fs from 'fs'
 
-const [,,command, ...rest] = process.argv
+const postinstallGlobals = {
```
Comment (chrisnojima) [inline desktop/npm-helper.js]: we now install this globally when you npm install

File: desktop/package.json [hunk 3]
```diff
@@ -66,14 +66,6 @@
     "electron-packager": "^5.1.0",
     "electron-prebuilt": "0.36.7",
     "electron-rebuild": "^1.1.3",
-    "eslint": "^1.10.3",
```
Comment (chrisnojima) [inline desktop/package.json]: everyone should rm -rf node_modules ; npm i

File: shared/actions/kbfs/index.desktop.js [hunk 4]
```diff
@@ -22,17 +22,17 @@ export function openInKBFS (path: string = ''): AsyncAction {
...
-          return Promise.reject(`There isn't exactly one kbfs client`)
+          return Promise.reject("There isn't exactly one kbfs client")
         }
 
         // Hacky Regex to find a mount point on windows matches anything like foobar:\ or K:\
         const kbfsPath = kbfsClients[0].argv.filter(arg => arg.search(/.*:\\?$/) === 0)[0]
 
         if (!kbfsPath) {
-          return Promise.reject(`Couldn't figure out kbfs path from argv`)
+          return Promise.reject("Couldn\'t figure out kbfs path from argv")
```
Comment (MarcoPolo) [inline shared/actions/kbfs/index.desktop.js]: the escape isn't needed here?

**Comment by MarcoPolo** [comment]
:+1:

**Comment by cjb** [comment]
:thumbsup:

**Pull request merged**